package.json

```diff
@@ -4,12 +4,21 @@
   "description": "Module to display featured film on Moodu",
   "main": "index.js",
   "scripts": {
+    "csvwrite": "--max_old_space_size=4096 server/csvwritepsql.js",
```

Comment:
Did you give this a try with different numbers for max_old_space_size? I'm curious how much of a difference it makes.

Reply:
This didn't help me. I did try it but The solution for me was to change the max allocation by vscode instead